appserver/tests/payara-samples/samples/concurrency/pom.xml

```diff
@@ -57,7 +57,7 @@
         <maven.compiler.target>8</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <failOnMissingWebXml>false</failOnMissingWebXml>
-        <jakartaee-api.version>9.1.0</jakartaee-api.version>
+        <jakartaee-api.version>10.0.0-SNAPSHOT</jakartaee-api.version>
```

Comment:
Is this property not redundant?  It's already defined at the top-level pom

Reply:
yes it is redundant, now removed